subs2srs.lua

```diff
@@ -70,6 +70,7 @@ local config = {
     secondary_field = "SentEng",
     audio_field = "SentAudio",
     image_field = "Image",
+    image_attributes = 'alt="snapshot"', -- Additional image attributes to add to Anki
```

Comment:
If you need to control image attributes, you could do this instead, in a more general way

```
image_template=[[<img alt="snapshot" src="%s">]]
```
Or
```
image_template=[[<img alt="snapshot" data-editor-shrink="true" src="%s">]]
```

Reply:
I like that idea, consider it done. Also added a brief warning in the example config.